deployment/kubernetes/helm/charts/clamav/templates/clamav-deployment.yaml

```diff
@@ -61,7 +61,7 @@ spec:
         #     initialDelaySeconds: 20
         #     timeoutSeconds: 2
       securityContext:
-        runAsUser: 1000
+        runAsUser: 0
```

Comment:
Running it as root to avoid the permission error on /var/log/clamd/clamf.conf

Reply:
Please keep this 1000

Reply:
change the folder path to /tmp/log